includes/class-sensei-course.php

```diff
@@ -193,9 +201,108 @@ public function highlight_menu_item( $submenu_file ) {
 	}
 
 	/**
-	 * Add custom navigation to the admin pages.
+	 * Add showcase courses upsell page.
+	 *
+	 * @since $$next-version$$
+	 * @internal
+	 * @access private
```

Comment:
I don't think we need `@access private` if we have `@internal`

Reply:
Updated in https://github.com/Automattic/sensei/pull/6648/commits/bb022255ec70e13c738c62430bad12688da5f5f2